lucene/analysis/common/src/java/org/apache/lucene/analysis/hunspell/SpellChecker.java

```diff
@@ -67,6 +75,14 @@ public boolean spell(String word) {
     return false;
   }
 
+  private boolean removeTrailingDots(String word) {
```

Comment:
Should it be called spellWithTrailingDots() or something? Right now it reads as if it had side effects... Maybe I'm paranoid though.